src/bootstrap/src/lib.rs

```diff
@@ -1915,6 +1915,11 @@ to download LLVM rather than building it.
         stream.reset().unwrap();
         result
     }
+
+    /// Checks if the given target is the same as the host target.
+    fn is_host_target(&self, target: &TargetSelection) -> bool {
+        &self.config.build == target
+    }
```

Comment:
Discussion: this isn't quite accurate I think, if we are under Canadian cross build scenarios? People do try to use `(build, host, target)` platforms where `build != host != target`, right? This is more "a given target tuple matches that of the build platform", right?